app/templates/PACKAGE/_DEPENDENCIES.json

```diff
@@ -1,53 +1,53 @@
 {
   "dependencies": {
-    "@fancyapps/fancybox": "latest",
+    "@fancyapps/fancybox": "3.5.3",
     "bootstrap": "latest",
     "jquery": "latest",
     "kv.cookieconsent": "1.0.0"
   },
   "devDependencies": {
-    "@babel/core": "7.0.1",
-    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "7.0.0",
-    "@babel/plugin-transform-modules-commonjs": "7.0.0",
+    "@babel/core": "7.2.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.2.0",
+    "@babel/plugin-syntax-dynamic-import": "7.2.0",
+    "@babel/plugin-transform-modules-commonjs": "7.2.0",
     "@babel/polyfill": "7.0.0",
-    "@babel/preset-env": "7.0.0",
+    "@babel/preset-env": "7.2.0",
     "@babel/register": "7.0.0",
-    "ansi-colors": "3.0.5",
-    "autoprefixer": "9.1.5",
-    "babel-eslint": "9.0.0",
+    "ansi-colors": "3.2.3",
+    "autoprefixer": "9.4.2",
+    "babel-eslint": "10.0.1",
     "babel-loader": "next",
-    "browser-sync": "2.24.7",
-    "cfonts": "2.2.1",
+    "browser-sync": "2.26.3",
+    "cfonts": "2.3.0",
     "cross-env": "5.2.0",
-    "cssnano": "4.1.0",
+    "cssnano": "4.1.7",
     "del": "3.0.0",
-    "eslint": "5.6.0",
+    "eslint": "5.10.0",
     "eslint-config-kraftvaerk": "0.0.5",
-    "fancy-log": "1.3.2",
-    "gulp": "next",
+    "fancy-log": "1.3.3",
+    "gulp": "4.0.0",
```

Comment:
Require fixed version of gulp. Please refer to https://github.com/kraftvaerk/generator-rammevaerk/issues/49.